packages/utils/package.json

```diff
@@ -51,5 +51,9 @@
     "reflect-metadata": "^0.1.13",
     "semver-lite": "0.0.6"
   },
-  "gitHead": "5526910fd5974d09e80b5ba6324043b4409dd179"
+  "gitHead": "5526910fd5974d09e80b5ba6324043b4409dd179",
+  "devDependencies": {
+    "@types/debug": "^4.1.7",
+    "debug": "^4.3.4"
```

Comment:
i think `debug` should be in dependencies or eventually peerDependencies, whereas `@types/debug` should remain in devdependencies